<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uiPriv</class>
 <widget class="QDialog" name="uiPriv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>88</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>319</width>
    <height>88</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>319</width>
    <height>88</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Zaproszenie</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../scc.qrc">
    <normaloff>:/3rdparty/images/r2_32.png</normaloff>:/3rdparty/images/r2_32.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonReject">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>96</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>OdrzuÄ‡</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonIgnore">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>96</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Ignoruj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAccept">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>96</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Akceptuj</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../scc.qrc"/>
 </resources>
 <connections/>
</ui>
